job-service-container/pom.xml

```diff
@@ -428,6 +444,7 @@
                                 <ports>
                                     <port>${webservice.adminport}:8080</port>
                                     <port>${webservice.https.adminport}:8443</port>
+                                    <port>${webservice.healthcheckport}:8081</port>
```

Comment:
You should also add an entry for the new property in the `use-default-fixed-ports` profile.